internal/release/dynatrace-otel-collector.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=Dynatrace OpenTelemetry Collector
+After=network.target
```

Comment:
I think we could add a `Documentation=` option, with a pointer to our docs. Not a blocker, but might be helpful.

Reply:
good suggestion! done